### PR TITLE
Add meetup#2 session info

### PR DESCRIPTION
**Pull request by hidetakanishihara**
I added contens for omni meetup#2, please review it.

### CODE REVIEW

File: content/ja/meetup/002/_index.md [hunk 1]
```diff
@@ -7,5 +7,28 @@ weight: -2
...
+  - title: "free5GCの性能測定してみた"
+    speaker: leon3298
+    docs: 
+      - "https://github.com/omni-jp/meetup/raw/master/"
+  - title: "[OAI](https://openairinterface.org/) v1.2.2にスマフォを繋げた結果"
+    speaker: Harada Masayuki
+    docs: 
+      - "https://github.com/omni-jp/meetup/raw/master/"
+  - title: "Generic VNFM OpenStack Tackerのご紹介"
+    speaker:Hiroshi Tsuji
```
Comment (t-matz) [inline content/ja/meetup/002/_index.md]: need a space following colon.
```suggestion
    speaker: Hiroshi Tsuji
```

File: content/ja/meetup/002/_index.md [hunk 2]
```diff
@@ -7,5 +7,28 @@ weight: -2
 #applicationForm: https://docs.google.com/forms/d/e/1FAIpQLSfPyOnXtg-nGFLD7H0tIMLD2J00URC0J9-IlwVmhCu_qU48hw/viewform
 meetupInfo: https://omni-jp.connpass.com/event/204562/
 presentations:
+  - title: "Opening"
+    speaker: Hidetaka Nishihara
+    docs: 
+      - "https://github.com/omni-jp/meetup/raw/master/"
```
Comment (t-matz) [inline content/ja/meetup/002/_index.md]: You can add documents info after uploading materials. Please check same things below.

```suggestion
#    docs: 
#      - "https://github.com/omni-jp/meetup/raw/master/"
```

**Review by t-matz — CHANGES_REQUESTED** [review]
Added comments.

File: content/ja/meetup/002/_index.md (hunk 1 above)
Comment (hidetakanishihara) [reply content/ja/meetup/002/_index.md]: Thanks! I fixed it , so please review it again.

File: content/ja/meetup/002/_index.md (hunk 2 above)
Comment (hidetakanishihara) [reply content/ja/meetup/002/_index.md]: Thanks! I added documents info, so please review it again.

**Review by t-matz — APPROVED** [review]

**Pull request merged**